features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html

```diff
@@ -176,7 +176,7 @@ <h2 id="TargetOperatingEnvironments2">
   <p>Most of the Eclipse SDK is "pure" Java code and has no direct dependence on the underlying operating system. The chief dependence is therefore on the Java Platform itself. Portions are
     targeted to specific classes of operating environments, requiring their source code to only reference facilities available in particular class libraries (e.g. J2ME Foundation 1.1, J2SE 1.4, Java
     5, etc).</p>
-  <p>In general, the 4.27 release of the Eclipse Project is developed on Java SE 11 VMs. As such, the Eclipse SDK as a whole is targeted at all modern, desktop Java VMs.</p>
+  <p>In general, the 4.28 release of the Eclipse Project is developed on Java SE 11 VMs. As such, the Eclipse SDK as a whole is targeted at all modern, desktop Java VMs.</p>
```

Comment:
Minimum required version is Java 17 now.

Reply:
Updated